tests/testthat/test-create_item.R

```diff
@@ -3,6 +3,7 @@
 test_that("simple item creation", {
   withr::with_tempdir({
     # set up "validation" infrastructure
+    writeLines(c("working_dir: vignettes"),".validation")
     dir.create("vignettes/validation", recursive = TRUE)
```

Comment:
why manual creation of the validation config and folder structure only? should have at least one case demonstrating that  `create_item` can play nicely with outputs of the `vt_use_` functions.

Reply:
when I wrote the tests I wanted to explicitly test just the create_item functionality. added test that uses `vt_use_validation_config()` and `vt_use_validation()` instead of the manual set up in 350b672